akvo/rest/serializers/project.py

```diff
@@ -74,6 +73,8 @@ class ProjectSerializer(BaseRSRSerializer):
     can_edit_enumerator_access = serializers.SerializerMethodField()
     program = serializers.SerializerMethodField()
     targets_at = TargetsAtField(choices=Project.TARGETS_AT_OPTION, required=False)
+    path = serializers.SerializerMethodField()
+    uuid = serializers.ReadOnlyField()
```

Comment:
I'm not sure about the need to expose the `path` and `uuid` attributes.

Reply:
Fixed in 03fab4f696663e8371d24aa781c8c63995a81e3f